app.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, render_template, request
+import sys
+
+sys.path.insert(0, 'src')
+
+from contacts_book import *
+from contact import * 
+
+app = Flask(__name__)
+
+contacts_book = ContactsBook()
+
+@app.route('/')
+def index():
+    return render_template('home.html', contacts = contacts_book.get_contacts())
+
+@app.route('/add', methods=['POST'])
+def addContact():
+    name = request.form['name']
```

Comment:
Not a big deal, but it feels like this responsibility (picking data from the `form` and creating a `Contact` with it) should not be at the controller level. 
Maybe the `Contact` constructor could take the `request.form` instead of a name and telephone? If that was the case, then your controller would be slimmer:

```
    contacts_book.add_contact(request.form)
    return render_template('home.html', contacts = contacts_book.get_contacts())
```

Reply:
Done.